scene/main/node.cpp

```diff
@@ -1070,7 +1107,7 @@ void Node::_add_child_nocheck(Node *p_child, const StringName &p_name) {
 	add_child_notify(p_child);
 }
 
-void Node::add_child(Node *p_child, bool p_legible_unique_name) {
+void Node::add_child(Node *p_child, bool p_legible_unique_name, int p_internal) {
```

Comment:
Additionally, I am uncertain how many times legible unique name is used (I think only the editor uses it), so it may be better to swap the order of the two arguments to make this one come second.